doc/nrf/app_bootloaders.rst

```diff
@@ -0,0 +1,63 @@
+.. _app_bootloaders:
+
+Bootloaders and Device Firmware Updates
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |NCS| enables developers to create applications that use multiple types of bootloaders.
+These types can also be combined for products that need a second-stage bootloader capable of providing device firmware updates to itself and the application, in the following ways:
+
+* Direct Device Firmware Updates - updates sent through a wired connection, like UART, or delivered connecting a flash device.
```

Comment:
wired connection does not mean the update will be direct
there can be in-place image substitution (what I think you have meant here) and a backgroud DFU where you first store the imaged obtained via USB or UART but complete the update process later on.

Reply:
I think the DFU should be split into:
* direct
* background
Transport used for DFU can be
* wired
* wireless (OTA)

Theoretically you can have direct DFU with wireless transport if you add stack to bootloader. In this case application could be erased.

Reply:
@pdunaj rephrased, please check the updated version

Reply:
This part looks great now :)